staging/src/k8s.io/apiserver/pkg/storage/testing/utils.go

```diff
@@ -357,3 +359,42 @@ func (s sortablePodList) Less(i, j int) bool {
 func (s sortablePodList) Swap(i, j int) {
 	s[i], s[j] = s[j], s[i]
 }
+
+func CreatePodPredicate(field fields.Selector, namespaceScoped bool, indexField []string) storage.SelectionPredicate {
```

Comment:
Happy to approve this part of the PR if you move it to a separate one.

Reply:
Extracting utils is not beneficial if they don't have different use-case than in this PR.